openshift-ci/Dockerfile.tools

```diff
@@ -10,7 +10,7 @@ ENV GIT_COMMITTER_EMAIL pipelines-dev@redhat.com
 
 RUN yum install epel-release -y
 
-RUN yum --enablerepo=centosplus install -y  \
```

Comment:
do we need to enable any other repo instead `centosplus` ?

Reply:
this works without any repo, so I just removed it